releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,23 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 8.0.0 - October 3rd, 2024
+### Breaking Changes
+* Increased the default maximum retry count when resolving Key Vault references and increased the minimum refresh interval to 1 minute. This helps prevent requests to Key Vault from being throttled and will reduce the chance of failing to resolve a secret due to momentary failures. [#589](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/589)
+* Removed .NET 7 as a target framework as .NET 7 is out of support. [#567](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/567)
+* The APIs `AzureAppConfigurationRefreshOptions.SetCacheExpiration` and `FeatureFlagOptions.CacheExpirationInterval` have been deprecated and will be removed in a future release. They are replaced with `AzureAppConfigurationRefreshOptions.SetRefreshInterval` and `FeatureFlagOptions.SetRefreshInterval`, respectively. This change does not affect functionality but aims to eliminate confusion regarding cache expiration implied by the previous API names. [#350](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/350)
+
+### Enhancements
+* **Variant Feature Flags**: This is the first stable release with support for the `variants`, `allocation`, and `telemetry` properties for feature flags added by version 4 of the [Microsoft.FeatureManagement](https://github.com/microsoft/FeatureManagement-Dotnet) library. This enables [experimentation with Azure App Configuration](https://learn.microsoft.com/en-us/azure/azure-app-configuration/concept-experimentation), which opens up possibilities for improving user experience or software functionality using data from your feature flags.
```

Comment:
- Telemetry should be called out separately from variant feature flags
- It's more than one MS.FM library
- The variant feature flags are not just for experimentation. I copied what I suggested for [FM release notes](https://github.com/Azure/AppConfiguration/pull/974/files) below.

**Variant Feature Flags and Telemetry**: This is the first stable release with support for the `variants`, `allocation`, `telemetry` etc. properties for feature flags added by version 4.0.0 of the [Microsoft.FeatureManagement.*](https://github.com/microsoft/FeatureManagement-Dotnet) libraries.  While variant feature flags can still be toggled on or off, they also allow for different configurations, ranging from simple primitives to complex JSON objects. Variant feature flags are particularly useful for feature rollouts, configuration rollouts, and feature experimentation (also known as A/B testing).